app-shells/command_not_found/command_not_found-0.0.1~git.recipe

```diff
@@ -39,9 +43,9 @@ BUILD()
 
 INSTALL()
 {
-	mkdir -p $binDir
+	install -d $binDir $postInstallDir
 	install -m 0755 haiku_cnf.py $binDir/command_not_found
 
-	mkdir -p $postInstallDir
-	mv install_cnf.py $postInstallDir
+	install -m 0755 install_cnf.py $postInstallDir
+	chmod +x install_cnf.py
```

Comment:
No need for the `chmod +x install_cnf.py`, since the `install -m 0755 install_cnf.py $postInstallDir` already takes care of that for us.

BTW, the chmod is not being applied on the installed script but on the one in the source directory, so it would have no effect in the hpkg.

Hmm, in case there was a TEST() then the chmod could have been required, but that's not the case here so you can safely drop the chmod (as long as you keep the `-m 0755` in the previous line.)